README.md

```diff
@@ -62,6 +62,22 @@ The above scrips will lint the js files while pushing to git. It will terminate
 ### With Errors
 <img src="screenshots/OutputWithErrors.gif" width="496" height="340" alt="With Erros">
 
+### Concurrent Tasks
+
+If you'd like to run your task concurrently, use the `concurrent` property like so:
```

Comment:
we need to convey that the main tasks will run in parallel and the subtasks will run in linear order. also users should know that this configuration is for sub tasks only.

Reply:
How about?

```suggestion
Tasks for a file group will by default run in linear order (eg. `"*.js": [ "jest", "eslint"]` will run jest first, then after it's done run eslint). 
If you'd like to run tasks for a file group concurrently instead (eg. jest and eslint in parallel), use the `concurrent` property like so:
```

Reply:
This looks cool 👍